services/app-api/src/domain-models/contractAndRates/contractRevision.ts

```diff
@@ -0,0 +1,122 @@
+import { ContractFormData } from './contractType'
+import { UpdateInfoTableWithUpdater } from '../../postgres/prismaTypes'
+import { UpdateInfoType } from '../HealthPlanPackageType'
+import {
+    ActuaryContact,
+    ContractDocument,
+    ContractRevisionTable,
+    ContractSupportingDocument,
+    FederalAuthority,
+    ManagedCareEntity,
+    StateContact,
+} from '@prisma/client'
+
+type ContractFormDataType = ContractRevisionTable & {
+    stateContacts: StateContact[]
+    addtlActuaryContacts: ActuaryContact[]
+    contractDocuments: ContractDocument[]
+    supportingDocuments: ContractSupportingDocument[]
+    managedCareEntities: ManagedCareEntity[]
+    federalAuthorities: FederalAuthority[]
+}
+function contractFormDataToDomainModel(
+    contractRevision: ContractFormDataType
+): ContractFormData {
+    return {
+        programIDs: contractRevision.programIDs,
```

Comment:
I'm a little confused why we aren't passing along the `id` `updatedAt` and `createAt` here. What was the thinking behind that?

Reply:
From in person sync with Jason: let's deal with this when we get to the createContract resolver.